torchbraid/odenet_apps.py

```diff
@@ -174,53 +153,37 @@ def eval(self,y,tstart,tstop,level,done,x=None):
     condition x. The level is defined by braid
     """
 
-    # this function is used twice below to define an in place evaluation
-    def in_place_eval(t_y,tstart,tstop,level,t_x=None):
-      # get some information about what to do
-      dt = tstop-tstart
-      layer = self.getLayer(tstart,tstop,level) # resnet "basic block"
-
-      #print(self.my_rank, ": FWDeval level ", level, " ", tstart, "->", tstop, " using layer ", layer.getID(), ": ", layer.linearlayer.weight[0].data)
-
-      if t_x==None:
-        t_x = t_y
-      else:
-        t_y.copy_(t_x)
-
-      q = dt*layer(t_x)
-      t_y.add_(q)
-      del q
-    # end in_place_eval
-
     # there are two paths by which eval is called:
```

Comment:
This comments is confusing now, and probably should be removed. As far as I see it, there is not anymore two path which call eval, right?